misk-hibernate/src/main/kotlin/misk/hibernate/EncryptedColumn.kt

```diff
@@ -0,0 +1,4 @@
+package misk.hibernate
+
+@Target(AnnotationTarget.FIELD)
+annotation class EncryptedColumn(val keyName: String)
```

Comment:
seems burdensome to have to include the key for every encrypted column. does the key change per column?

Reply:
It could.
I can imagine a scenario where you'd want to encrypt different columns with different keys, depending on their secrecy, what other systems might need access to that data and so on.

Also, having this configurable decouples the `misk-crypto` and `misk-hibernate` configuration.

Reply:
do you think that's the common case though? should there be a default key if the app use the same key for every column?

Reply:
That's a good point.
I think having a default key name could be really cool in terms of user experience.

Reply:
it's not really clear this keyName should match the keyName in the crypto config. some javadoc would be helpful

Reply:
Done

Reply:
Regarding a default key and having a single key for all columns - I think maybe we shouldn't do that.
Having a default key implies there's a key somewhere that's not managed by the owner of the app.
As for specifying a single key for all columns, it' could be a nice improvement.
But, assuming you usually encrypt one or 2 columns in a table, if at all, I'd go with the explicit approach of specifying the key name for each of these columns.